File: RemObjects-C#/MainWindowController.cs
using AppKit;
using JavaScriptCore;
using WebKit;

namespace RemObjectsC {
	[IBObject]
	public class MainWindowController : NSWindowController {
		[IBOutlet]
		public WebView myWebView { get; set; }
		
		public override instancetype init() {
			this = base.initWithWindowNibName("MainWindowController");
			if (this != null) {
				// Custom initialization
			}
			return this;
		}

		public override void windowDidLoad() {
			base.windowDidLoad();
			
			myWebView.frameLoadDelegate = new MyFrameLoadDelegate();
			
			NSUserDefaults.standardUserDefaults.setBool(true) forKey("WebKitDeveloperExtras");
			NSUserDefaults.standardUserDefaults.synchronize();
			
			NSURLRequest req = new NSURLRequest withURL(new NSURL withString("http://demo.desktopjavascript.dev:8888/"));
			myWebView.mainFrame.loadRequest(req);
		}
	}
}

File: RubyMotion/app/main_window_controller.rb
class MainWindowController < NSWindowController
  def initialize(window)
    initWithWindow(window)
    self.loadWindow
  end

  def loadWindow
    # windowWillLoad
    super
    windowDidLoad
  end

  def windowNibName
    "MainWindowController"
  end

  def windowDidLoad
    super
    self.window.title = NSBundle.mainBundle.infoDictionary['CFBundleName']
    NSUserDefaults.standardUserDefaults.setBool(true, forKey: "WebKitDeveloperExtras")
    NSUserDefaults.standardUserDefaults.synchronize
    @myWebView = WebView.alloc.initWithFrame(NSMakeRect(0, 0, 540, 400))
    @myWebView.setAutoresizingMask(NSViewMinXMargin|NSViewMaxXMargin|NSViewMinYMargin|NSViewMaxYMargin|NSViewWidthSizable|NSViewHeightSizable)

    @myWebView.frameLoadDelegate = MyFrameLoadDelegate.new
    @myWebView.mainFrame.loadRequest(NSURLRequest.requestWithURL(NSURL.URLWithString("http://demo.desktopjavascript.com")))

    self.window.contentView.addSubview(@myWebView)
    self.window.orderFrontRegardless
  end
end